pkg/services/apiserver/storage/entity/utils.go

```diff
@@ -73,7 +73,7 @@ func entityToResource(rsp *entityStore.Entity, res runtime.Object, codec runtime
 		originTime := time.UnixMilli(rsp.Origin.Time).UTC()
 		grafanaAccessor.SetOriginInfo(&utils.ResourceOriginInfo{
 			Name: rsp.Origin.Source,
-			Key:  rsp.Origin.Key,
+			Hash: rsp.Origin.Key,
```

Comment:
If we're renaming Key to Hash we should probably do it everywhere, otherwise it's just going to get confusing. especially when in some places we've changed Key to Path and in others to Hash

Reply:
hymmm -- I'm going to move the "Key" > "Hash" to its own PR -- looks like the mixed/mistaken assumptions on what it means and how it is used are deeper than thought